### DISCUSSION TITLE
SNMP on Prometheus Error

### DISCUSSION THREAD

**Discussion by hndmkmhhck** [prometheus/snmp_exporter#1089] [General]
Hi Guys,

Seeking for help!

I have this error on my Prometheus which uses snmp_exporter:
![image](https://github.com/prometheus/snmp_exporter/assets/151911745/9c27601e-ebbb-4ef9-adb6-5e4e7b8d9124)

The goal is to see SNMP metrics from Fortigate and Aruba switches into Prometheus, but to no success, badly need help. Thank you!

**Comment by SuperQ** [comment]
`auth` is only allowed once.

**Reply by hndmkmhhck** [reply]
![image](https://github.com/prometheus/snmp_exporter/assets/151911745/146024bb-55b6-447c-bffd-673ea8d9aba7)

The reason it has 2 auths because the version 2 is for the HP switches, while the version 3 is for the Fortinet-FW

The picture is my prometheus' snmp config, may you kindly look at it? 

because i actually dont have any knowledge on creating Grafana and Prometheus, not even the slightest.
